code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -621,6 +621,22 @@
 		W.charges = W.max_charges
 	update_icon()
 
+/obj/item/storage/belt/wands/fireballs
+	name = "infernal belt"
+	desc = "\"Use Fireball and only Fireball. Nothing but fireball. Just Fireball. <b>Just Fireball.</b>\""
+	icon_state = "militarybelt"
+	item_state = "military"
+	can_hold = list(
+		/obj/item/gun/magic/wand/fireball
+	)
+
+/obj/item/storage/belt/wands/fireballs/populate_contents()
+	for(var/count in 1 to storage_slots)
+		var/obj/item/gun/magic/wand/fireball/W = new /obj/item/gun/magic/wand/fireball(src)
+		W.max_charges = initial(W.max_charges)
```

Comment:
Why are we resetting this to something that was just initialized?

Reply:
because in new, it sets the charges to something random. look at the wand belt and nanny bag